explainability-core/src/main/java/org/kie/trustyai/metrics/drift/image/ImageDrift.java

```diff
@@ -0,0 +1,102 @@
+package org.kie.trustyai.metrics.drift.image;
+import org.kie.trustyai.explainability.model.Dataframe;
+import org.kie.trustyai.explainability.model.Type;
+import org.kie.trustyai.explainability.model.Value;
+
+import java.awt.image.BufferedImage;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Jensen-Shannon divergence to calculate image data drift.
+ * See <a href="https://en.wikipedia.org/wiki/Jensen%E2%80%93Shannon_divergence">Jensen–Shannon divergence</a>
+ */
+public class ImageDrift {
+     /**
+     * Converts an image to Dataframe whose columns are "Red", "Green", "Blue" by extracting the associated RGB values.
+     *
+     * @param image An image.
+     * @return A Dataframe.
+     */
+    public static Dataframe getDataframe(BufferedImage image){
+        int width = image.getWidth();
+        int height = image.getHeight();
+        Dataframe dataframe = new Dataframe();
+        List<Value> redCol = new ArrayList<>();
+        List<Value> greenCol = new ArrayList<>();
+        List<Value> blueCol = new ArrayList<>();
+
+        int totalPixels = width * height;
+        for (int i = 0; i < width; i++){
+            for (int j = 0; j < height; j++){
+                int pixel = image.getRGB(i, j);
+                redCol.add(new Value ((pixel >> 16) & 0xff / totalPixels));
+                greenCol.add(new Value ((pixel >> 8) & 0xff / totalPixels));
+                blueCol.add(new Value ((pixel) & 0xff / totalPixels));
+            }
+        }
+        dataframe.addColumn("Red", Type.NUMBER, redCol);
+        dataframe.addColumn("Green", Type.NUMBER, greenCol);
+        dataframe.addColumn("Blue", Type.NUMBER, blueCol);
+
+        return dataframe;
+    }
+
+     /**
+     * Calculates the Jenson-Shannon divergence between two pixel arrays.
+     *
+     * @param p1 The reference pixel array.
+     * @param p2 The hypothesis pixel array.
+     * @return The Jenson-Shannon divergence.
+     */
+    public static double jensonShannonDivergence(double[] p1, double[] p2){
+        double[] m = new double[p1.length];
+        for (int i = 0; i < p1.length ; i++){
+            m[i] = (p1[i] + p2[i]) / 2.0;
+        }
+        return (KLDivergence(p1, m) + KLDivergence(p2, m)) / 2.0;
+    }
+
+     /**
+     * Calculates the Kullback-Leibler divergence between 2 pixel arrays.
+     *
+     * @param p1 The reference pixel array.
+     * @param p2 The hypothesis pixel array.
+     * @return The Kullback-Leibler divergence.
+     */
+    public static double KLDivergence(double[] p1, double[] p2){
```

Comment:
KLDivergence between two arrays might be useful on its own.
I would move this to a more general utility class for re-use and call that class here.